README.md

```diff
@@ -28,4 +28,74 @@ Please note that you need to change the template path to load a customized
 template, adding a customized template with the default path and name in your
 playbook structure won't work.  E.g. Changing the "atom/config/config.php"
 to "myatom/config/config.php" will load a template file at that relative
-path in your playbook directory structure.
\ No newline at end of file
+path in your playbook directory structure.
+
+## Deploy multiple sites on the same host
+
+To deploy multiple AtoM sites on the same host, it is possible to have
+an arrangement as follows:
+
+```
+.
+|-- host_vars
+|   `-- atomhost
+|       |-- vars.yml       => variables common for all sites
+|       |-- ...
+|-- sites
+|   |-- atomsite1.yml      => variables for site1
+|   |-- atomsite2.yml      => variables for site2
+|   |-- ...
+|-- playbook-atom.yml      => AtoM deployment playbook
+|-- ansible.cfg
+|-- hosts
+|-- ... 
+```
+
+Where `atomsite1.yml`, `atomsite2.yml`, ... define role variables that
+need to be different for site1, site2,... 
+
+As a minimum, the variable `atom_path` must be different for each site. 
+The basename of `atom_path` is used to create identifiers that need to be 
+different for each site, such as php pool and worker names (the basename
+is the last component of the path, e.g., if `atom_path` is 
+"/usr/share/nginx/atom" then the basename is "atom")
+
+For example, if deploying AtoM production and test sites, we could 
+define the production site variables in `atomsite1.yml`:
+
+```
+atom_path: "/usr/share/nginx/atom"
+atom_config_db_name: "atom_prod"
+atom_config_db_username: "atomuser_prod"
+atom_config_db_password: "{{ vault_atomuser_prod_pass }}"
+atom_es_index: "atom_prod"
+```
+
+And the test site variables in `atomsite2.yml`:
+
+```
+atom_path: "/usr/share/nginx/atom-test"
+atom_config_db_name: "atom_test"
+atom_config_db_username: "atomuser_test"
+atom_config_db_password: "{{ vault_atomuser_test_pass }}"
+atom_es_index: "atom_test"
+```
+
+To deploy AtoM dependencies and the first site on the host:
+```
+ansible-playbook playbook-atom.yml -l atomhost -e @sites/atomsite1.yml
+```
+
+Then to deploy the second site:
+```
+ansible-playbook playbook-atom.yml -l atomhost -e @sites/atomsite2.yml -t atom-site
+```
+
+The `-e @<file>` (extra vars) option makes ansible use the variables defined in the 
+specified file in addition to the variables defined in host_vars and the playbook
+(with the values assigned as extra vars taking precedence over assignments
+done elsewhere)
+
+The `-t atom-site` option makes ansible execute only the tasks tagged
```

Comment:
I had to run with ```-t percona, atom-site```  in order for the mysql db to be created

Reply:
The tag "percona" references to another role (ansible-percona), I think it's better just to include the tags relevant for this ansible role in this example, to avoid confusion.

Reply:
Still, some note regarding the need of the mysql db to be already created won't hurt